docs/source/installation.rst

```diff
@@ -0,0 +1,22 @@
+Installation
+============
+
+Conu is now in initial phase. To try it out use following commands:
+
+Clone repository:
+
+.. code-block:: bash
+
+   git clone https://github.com/fedora-modularity/conu
+
+Install requirements
+
+.. code-block:: bash
+
+   sh -c conu/requirements.sh
+
+Add conu directory to PYTHONPATH variable, so it can be imported
+
+.. code-block:: bash
+
+   export PYTHONPATH=$PYTHONPATH:${PWD}/conu
```

Comment:
I installed it using `pip3 install --user .`, what do you think?

Reply:
I think that it looks nicer. It should now be fixed.